app/Models/GroupMembership.php

```diff
@@ -19,6 +20,14 @@ class GroupMembership extends Pivot
 		'role' => GroupRole::class,
 	];
 	
+	protected static function booted()
```

Comment:
Would it make sense to only do this for admins, not all users?

Reply:
I guess it just seems easier to do it this way, and it's safer with very minimal overhead…